mantle/platform/qemu.go

```diff
@@ -306,6 +306,41 @@ func (inst *QemuInstance) SwitchBootOrder() (err error) {
 	return nil
 }
 
+// RemoveBlockDevice deletes the primary device from a qemu instance
+// and sets the seconday device as primary.
+func (inst *QemuInstance) RemoveBlockDevice() error {
```

Comment:
This doesn't actually seem to be deleting the primary block device, only changing the bootindex.

Reply:
ah, thanks for catching that. I've changed the previous implementation but forgot to change the function name